docs/runbooks/KubeVirtDeprecatedAPIsRequested.md

```diff
@@ -0,0 +1,33 @@
+# KubeVirtDeprecatedAPIsRequested
```

Comment:
@sradco 
I just noticed that the title and file name are a little weird because it should be "...APIIsRequested" or "...ApiIsRequested". It is missing an "i". 

For the sake of consistency with the rest of the runbooks, would it be possible to change the title/file name to `KubeVirtDeprecatedAPIRequested`?

Reply:
It was meant to be the plural of `API`, not that a deprecated API is requested, at least in the title. But it might make sense to change it to the singular.

Reply:
I admit that option did not occur to me. If you can change it to the singular, that would make it clearer. Please tag me in the PR so that I can update the openshift/runbooks file and the openshift/openshift-docs PR.